src/java/org/apache/cassandra/io/sstable/AbstractSSTableSimpleWriter.java

```diff
@@ -110,6 +112,39 @@ private static SSTableId getNextId(File directory, final String columnFamily) th
         }
     }
 
+    /**
+     * Returns the partition update builder for the given key
```

Comment:
Returns the partition update builder for the given key.

Missing a period (if you think it's needed), just align with the description of the method below